.github/workflows/linters.yaml

```diff
@@ -1,9 +1,6 @@
 name: "linters"
 on:
-  push:
```

Comment:
No longer doing this on push to avoid `workflow_run` being activated when there's no PR. Hopefully we shouldn't ever be in a position where we merge when important linters failed.